packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -69,10 +77,14 @@ export const getFinclusiveBankAccounts = async ({
       },
     },
   })
+  if (!response.ok) {
+    throw new Error(`IHL GET /account/bank-account failure status ${response.status}`)
+  }
+  const { bankAccounts } = await response.json()
+  return bankAccounts
```

Comment:
nice ✅

Reply:
moving the return type assignments and the response status checks to this layer is a good improvement